src/components/court-order-poa/CourtOrderPoa.vue

```diff
@@ -45,9 +45,6 @@ export default class CourtOrderPoa extends Vue {
     courtNumRef: FormIF
   }
 
-  /** Prompt the validations. Used for global validations. */
-  @Prop({ default: false }) readonly autoValidation!: boolean
```

Comment:
Good riddance!

Reply:
I swear there was some archaic validation requirement that spawned this blasphemous code 👀 
Good riddance indeed 😆